stable/node-local-dns/README.md

```diff
@@ -12,37 +12,11 @@ Further documentation is [here](https://kubernetes.io/docs/tasks/administer-clus
 
 This helm chart works for both kube-proxy setups (iptables or ipvs).
 
-## How to install this chart
```

Comment:
The PR is good but this templated text needs to stay. Perhaps you are not generating the templated READMEs it from repo root?

Reply:
Hey I've run `docker run --rm -v "${PWD}:/helm-docs" jnorwood/helm-docs:v1.11.3 --template-files ./ci/README.md.gotmpl` from this dir `stable/node-local-dns`. 
I have to run it from the root ?